lib/ui/painting.dart

```diff
@@ -1591,6 +1605,21 @@ enum ImageByteFormat {
   /// image may use a single 8-bit channel for each pixel.
   rawUnmodified,
 
+  /// Raw extended range RGBA format.
+  ///
+  /// Unencoded bytes, in RGBA row-primary form with straight alpha, 32 bit
```

Comment:
I think this matches the encoding of the dart:ui Color class, right?

Reply:
The dart:ui Color class is using integer values for components, this is float32.